ocean-client/src/programs/vault-maxi-program.ts

```diff
@@ -35,6 +35,58 @@ export enum VaultMaxiProgramTransaction {
   StableArbitrage = 'stablearbitrage',
 }
 
+export enum LogLevel {
```

Comment:
the log level has nothing to with vault-maxi program, right? Maybe we should move it to a shared (or utils) folder, as every program is using it

Reply:
basically it belongs to telegram, right?